### PR TITLE
fixes issue #231 -  changed  developer's website link to author name

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #231. Changed the label of developer's website link
to developer's name.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link : http://loklakapps.surge.sh/details.html?q=sentimentVisualizer

@kavithaenair @SKrPl @Achint08 @hemantjadon open for review

File: details.html [hunk 1]
```diff
@@ -177,7 +177,7 @@ <h5 class="info-header">
                   </div>
                   <div class="info-body">
                     <span ng-if="appData.author.url !== undefined && appData.author.url !== ''">
-                      <a href="{{appData.author.url}}"> Visit developer's website </a>
+                      <a href="{{appData.author.url}}"> {{appData.author.name}} </a>
                     </span><br>
                     <span ng-if="appData.author.email !== undefined && appData.author.email !== ''">
                       <a href="mailto:{{appData.author.email}}"> Send an email </a>
```
Comment (vibhcool) [inline details.html]: how about link name be `contact me/author` at instead of `Send an email`

**Comment by djmgit** [comment]
http://loklakapps.surge.sh/details.html?q=sentimentVisualizer

@vibhcool made changes

**Comment by vibhcool** [comment]
yep, `glyphicon-envelope` would look better!

**Comment by djmgit** [comment]
@singhpratyush @vibhcool used envelope glyphicon

**Pull request merged**